test/image_test.py

```diff
@@ -1063,7 +1063,7 @@ def get_hash(img: Image) -> str:
         spec_file=test_dir / "supports" / "test-conda-environment.yml",
         channels=["conda-forge", "my-channel"],
     )
-    assert get_hash(img) == "d9d4c9fe24769ce587877b9752a64486e8f7d8520731110bd2fa666de82f43fd"
+    assert get_hash(img) == "f8701ce500d6aa1fecefd9c2869aef4a13c77ab03925333c011d7eca60bbf08a"
```

Comment:
This isn't a real change to the Image recipe, but the support `test-conda-environment.yml` support file we use had `python_version: 3.8`, which I've updated to avoid other issues.